Updating CONTRIBUTING.md (#775)
vtex/faststore: CONTRIBUTING.MD

1. After acquiring the necessary approval, rebase and squash-merge your PR in master

> 2. A bot will commit on master releasing your changes. In the mean time, go grab a coffe ☕

Reviewer: Typo: `meantime` and `coffee`

Author: Oops

> 2. A bot will commit on master releasing your changes. In the mean time, go grab a coffe ☕

Reviewer 2: ```suggestion
2. A bot will commit on master releasing your changes. In the meantime, go grab a coffee ☕
```

> If the bot isn't working or you want to release it manually, use the following command to clean any artifact and release a clean build:
>

> > `git clean -fdx && yarn && lerna run build && lerna publish minor --yes`

Reviewer: Is it ok to release minor version here?

Reviewer 3: patch!!

3. Update `package.json` of the `storecomponents` with:
```
yarn upgrade -L --scope @vtex
```

4. Stage, commit and push your changes with:
```
git add . && git commit -m "upgrade deps" && git push
```

5. Rebase with master
```
git rebase origin/master && git push --force
```

> 6. Wait for the bots to approve the PR.

Reviewer 4: I would rewrite it to something like that,
```suggestion
6. Wait for the PR checks to have been passed
```
